src/GetDocumentInsider/GetDocumentInsider.csproj

```diff
@@ -5,14 +5,22 @@
     <IsPackable>false</IsPackable>
     <OutputType>Exe</OutputType>
     <RootNamespace>Microsoft.Extensions.ApiDescription.Client</RootNamespace>
-    <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.0;netcoreapp2.1;netcoreapp2.2;net461</TargetFrameworks>
   </PropertyGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'net461'">
     <Reference Include="Microsoft.CSharp" />
   </ItemGroup>
 
-  <ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'net461' OR '$(TargetFramework)' == 'netcoreapp2.0'">
     <PackageReference Include="Microsoft.AspNetCore.TestHost" Version="$(MicrosoftAspNetCoreTestHost20PackageVersion)" />
   </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.1'">
```

Comment:
Not sure I fully understand this - TestHost is netstandard2.0, why do we need to cross compile the insider rather than multi-target?

Reply:
I agree everything is .NET Standard 2.0 from the insider down. The variance here is about the dependencies e.g. differences between Microsoft.AspNetCore.Hosting v2.x and v2.y. For example, moving from Microsoft.AspNetCore.TestHost 2.0.3 to 2.1.1 adds a dependency on System.IO.Pipelines. Basically, it's safest to be consistent with the web site's TFM.

However, not every web site targets .NET Core App 2.0. Multi-targeting is about being consistent w.r.t. the lower-level .NET Core dependencies.